### PR TITLE
DOMA-3450 added amplitude in callcenter

### PR DESCRIPTION
**Pull request by nomerdvadcatpyat**

### CODE REVIEW

File: apps/condo/domains/common/components/TrackingContext.tsx [hunk 1]
```diff
@@ -153,9 +154,17 @@ const useTracking: IUseTracking = () => {
...
     const router = useRouter()
 
+    const organizationId = get(organization, 'id', null)
+    const { objs: organizationLinks } = OrganizationLink.useObjects({
+        where: {
+            to: { id: organizationId },
+        },
+    })
+    const relatedFromOrganizationNames = organizationLinks.map(link => link.from.name)
+
```
Comment (AntonAL) [inline apps/condo/domains/common/components/TrackingContext.tsx:167]: Does these changes belong to the task?

**Review by AntonAL — COMMENTED** [review]
> Submodule [.helm](https://github.com/open-condo-software/infrastructure-condo) updated [from 52eb4d to 7aca57](https://github.com/open-condo-software/infrastructure-condo/compare/52eb4ded81af1505e6d19cdb2366907361d72cd7...7aca57ed9ba0b2669857172c2169c85e7c52bd44)
Does added new key have confidential values in `token` property?
```
    tracking_config:
      _default: '{}'
      production: '{"domains": {"ticket": ["create", "update", "id"]}, "trackers": {"amplitude": {"token": "...", "config": {}}}}'
      development: '{"domains": {"ticket": ["create", "update", "id"]}, "trackers": {"amplitude": {"token": "...", "config": {}}}}'
```

**Comment by AntonAL** [comment]
What coverage (how much places/components/events) of callcenter app by Amplitude is supposed to be implemented in this task?

**Review by akarjakin — APPROVED** [review]

**Comment by nomerdvadcatpyat** [comment]
> > Submodule [.helm](https://github.com/open-condo-software/infrastructure-condo) updated [from 52eb4d to 7aca57](https://github.com/open-condo-software/infrastructure-condo/compare/52eb4ded81af1505e6d19cdb2366907361d72cd7...7aca57ed9ba0b2669857172c2169c85e7c52bd44)
> 
> Does added new key have confidential values in `token` property?
> 
> ```
>     tracking_config:
>       _default: '{}'
>       production: '{"domains": {"ticket": ["create", "update", "id"]}, "trackers": {"amplitude": {"token": "...", "config": {}}}}'
>       development: '{"domains": {"ticket": ["create", "update", "id"]}, "trackers": {"amplitude": {"token": "...", "config": {}}}}'
> ```

This is ok, we are sending the token from the client

**Pull request merged**